paper-icon-button.html

```diff
@@ -135,6 +139,23 @@
        */
       icon: {
         type: String
+      },
+
+      /**
+       * Specifies the alternate text for the button, for accessibility.
+       */
+      alt: {
+        type: String,
+        observer: "_altChanged"
+      }
+    },
+
+    _altChanged: function(newValue, oldValue) {
+      var label = this.getAttribute('aria-label');
+
+      // Don't stomp over a user-set aria-label.
+      if (!label || oldValue == label) {
```

Comment:
Yeah, I think not caring about this case is fair - probably arguable that if a user sets `aria-label` to be the same as `alt` and then sets `alt` to something different, there's a good chance that they simply "forgot" to update `aria-label` at that point. So this LGTM.